go.mod

```diff
@@ -337,3 +337,5 @@ replace (
 )
 
 replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.golang v1.3.6-0.20220106130409-e28a1db639f8
+
+replace github.com/dapr/components-contrib => ../components-contrib
```

Comment:
We do not use this way, please implement it in components-contrib repo, after it merged, update components-contrib version here.